### PR TITLE
Feature: show issue assignee on project board

### LINKED ISSUE #15026: Feature request: show issue assignee on project board
- Gitea version (or commit ref): 1.13.2
- Git version: 1:2.20.1-2+deb10u3
- Operating system: debian 10
  
  
  
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Log gist:

## Description

Please, add additional info on projects page.
I am ready to help as much as I can.
Lower is an attachment of Github implemented this.

## Screenshots

image

### LINKED ISSUE #18326: Display assigned user in Kanban board
### Feature Description

The Kanban boards within Gitea should display the assigned user on each card. If an issue is unassigned, then this spot should be left blank or perhaps have some indicator that the issue is unassigned. Currently, the card displays the author of the issue, but not the developer working on the issue. My team uses Kanban boards as a task tracker, so it is much more useful to know who is working on the task, not who asked for the task to be done. Trello supports this feature by displaying the avatar (or initials) of the assigned user in the bottom-right corner of the card.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by rogerluo410**
Add assignee and poster on project board

Feature request issue:
[](url)https://github.com/go-gitea/gitea/issues/15026

UI:
WechatIMG1124

### CODE REVIEW

**Comment by rogerluo410** [comment]
WechatIMG1140

**Review by pat-s — APPROVED** [review]

**Comment by lunny** [comment]
It didn't show anything from my local test

image

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -2996,13 +2996,18 @@ tbody.commit-list {
 .board-column > .cards {
   flex: 1;
   display: flex;
-  flex-direction: column;
+  // flex-direction: column;
```
Comment (zeripath) [inline web_src/less/_repository.less]:
```suggestion
```

**Comment by rogerluo410** [comment]
> It didn't show anything from my local test
> 
> image

Did you set assignee ?

**Comment by lunny** [comment]
Please resolve the conflicts.

**Review by wxiaoguang — APPROVED** [review]
Merged, refactored and tested.

image

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**